### PR TITLE
Rs 2023.04.29

### PR DESCRIPTION
**Pull request by rschlaikjer**
D9 flowchart

### CODE REVIEW

File: system_strings/sysmes_text.en [hunk 1]
```diff
@@ -4917,167 +4982,269 @@ I feel her blatant animosity^and bloodlust. I recognize^her as an enemy. I'll lu
...
+// @flowchart_descr
+This is messed up. I want to part in it.
+ But Noel isn't satisfied at all by my refusal.
 }
 [sha:74a545322db885f4f945b43d9309406b93045bb5]{
 -- ノエルとビルを出て、路地裏へ。^彼女は自分の反応を愉しむように^シエルの過去を語り始める……。
-ノエルとビルを出て、路地裏へ。^彼女は自分の反応を愉しむように^シエルの過去を語り始める……。
+// @flowchart_descr
+Noel and I exit the building into a back alley.
+ She starts to tell me about Ciel's past, delighting in my reactions.
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:5167]: >...into the back alley below.
>...

File: system_strings/sysmes_text.en [hunk 2]
```diff
@@ -4917,167 +4982,269 @@ I feel her blatant animosity^and bloodlust. I recognize^her as an enemy. I'll lu
...
+I return to my room in low spirits.
+ Senpai issued me a warning, but I made a promise with Arcueid.
+ I can't make up my mind between the two...
 }
 [sha:853098cc87b3732c8375f362476529c8bd00d367]{
 -- 約束の時間だ。^アルクェイドにもう一度会って、^自分の気持ちを確かめよう。
-約束の時間だ。^アルクェイドにもう一度会って、^自分の気持ちを確かめよう。
+// @flowchart_descr
+The promised time has arrived.
+ I'll meet up with Arcueid one more time, and confirm how I really feel.
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:5180]: It's more like this. 約束 here just means his promise with Arcueid.
>The clock hits the time of my promise with Arcueid.

The follow up is from Offset 23361
>I'll meet up with her one more time, and confirm how I really feel.

might wanna change Offset 23361 too though

File: system_strings/sysmes_text.en [hunk 3]
```diff
@@ -4917,167 +4982,269 @@ I feel her blatant animosity^and bloodlust. I recognize^her as an enemy. I'll lu
...
 }
 [sha:0254e07a578acf4d16cf7b673be4c3c9db860126]{
 -- このままではまずい。^ここははっきりと言い返して、^シエル先輩にお帰り願おう。
-このままではまずい。^ここははっきりと言い返して、^シエル先輩にお帰り願おう。
+// @flowchart_descr
+I can't let things carry on like this.
+ I need to be blunt and ask Ciel-senpai to go back home.
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:5186]: the latter sentence is partially based on Offset 23483 so add that as a comment (don't need to change anything other than that)

File: system_strings/sysmes_text.en [hunk 4]
```diff
@@ -4917,167 +4982,269 @@ I feel her blatant animosity^and bloodlust. I recognize^her as an enemy. I'll lu
...
+Senpai told me I could do as I wish.
+ Now's the time to be logical, and ask Ciel-senpai to go back home.
 }
 [sha:9bf5309044ffd5661bf78f1c55e91c7dd36eacd1]{
 -- 公園に到着してしまった。^竜虎相打つ。^最悪の事態は容赦なく訪れる。
-公園に到着してしまった。^竜虎相打つ。^最悪の事態は容赦なく訪れる。
+// @flowchart_descr
+For better or worse, I've arrived at the park.
+ Now comes the clash of the titans.
+ Things are rapidly going south.
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:5199]: >...
>...
>Things couldn't possibly have gotten any worse.

File: system_strings/sysmes_text.en [hunk 5]
```diff
@@ -4917,167 +4982,269 @@ I feel her blatant animosity^and bloodlust. I recognize^her as an enemy. I'll lu
...
 }
 [sha:b69f7eb5d3de09b8f6774926152c6fb3d4c8e2a7]{
 -- 激突する代行者と吸血鬼。^トップランカーの戦いが、^ここに。
-激突する代行者と吸血鬼。^トップランカーの戦いが、^ここに。
+// @flowchart_descr
+The clash between Executor and vampire commences.
+ I'm witnessing a battle for the ages.
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:5205]: >...
>I stand witness to a battle for the ages.

File: system_strings/sysmes_text.en [hunk 6]
```diff
@@ -4917,167 +4982,269 @@ I feel her blatant animosity^and bloodlust. I recognize^her as an enemy. I'll lu
...
 }
 [sha:d7a92a84ed79beb1269075e32635b92eb0bcad84]{
 -- 伯仲している二人の戦闘能力。^その均衡は徐々に崩れてゆき、^そして、ついに――
-伯仲している二人の戦闘能力。^その均衡は徐々に崩れてゆき、^そして、ついに――
+// @flowchart_descr
+Their abilities start out evenly matched.
+ But that balance slowly crumbles, until suddenly――
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:5211]: 'start out' seems redundant given the next sentence.
is it really sudden if it happens slowly?
>Their abilities are evenly matched.
>But that balance slowly crumbles, until at last——

File: system_strings/sysmes_text.en [hunk 7]
```diff
@@ -4917,167 +4982,269 @@ I feel her blatant animosity^and bloodlust. I recognize^her as an enemy. I'll lu
...
 }
 [sha:e27c779151a8c6d0b06ac6b0961bf7b4fb194e7b]{
 -- このまま見過ごせる筈がない。^二人の間に割って入り、^アルクェイドと対峙する―――
-このまま見過ごせる筈がない。^二人の間に割って入り、^アルクェイドと対峙する―――
+// Cribbed from 23869.
+// @flowchart_descr
+I'm not going to just sit back and watch this insanity unfold!
+ I plant myself between the two of them, and confront Arcueid――
```
Comment (Petrikow) [inline system_strings/sysmes_text.en:5218]: Remove dash

**Review by Petrikow — COMMENTED** [review]
doing these feels like torture

File: system_strings/sysmes_text.en [hunk 8]
```diff
@@ -2645,167 +2645,232 @@ The lord has heard your prayer
...
-Morning greetings.
+// @flowchart_title
+Morning Greetings
 }
 [sha:ba3aaad7411d683f68ea41b90f575e4fd820a644]{
 -- 濡れた保健室
-The steamy infirmary.
+// Presumably this is love affair imi
+// @flowchart_title
+Steamy Infirmary
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply system_strings/sysmes_text.en:2754]: Hmm, I was thinking [hot and] steamy / keeping the wetness of 濡れ?

File: system_strings/sysmes_text.en [hunk 9]
```diff
@@ -2645,167 +2645,232 @@ The lord has heard your prayer
...
+// https://www.youtube.com/watch?v=2atkj_KWLl0
+// @flowchart_title
 Communication Breakdown
 }
 [sha:5614e6af15b57014467beabfec1621d16c3fc458]{
 -- 届いたメール
-I receive a text.
+// https://www.youtube.com/watch?v=dudJjUU9Nhs
+// @flowchart_title
+You've Got Mail
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply system_strings/sysmes_text.en:2784]: Yeah I did this mainly on the cuteness of the reference. If it's too much I can just revert.

File: system_strings/sysmes_text.en [hunk 10]
```diff
@@ -4917,167 +4982,269 @@ I feel her blatant animosity^and bloodlust. I recognize^her as an enemy. I'll lu
...
+ I'm worried about the look she had on her face last night.
+ Have her wounds not fully healed yet?
 }
 [sha:d05659f5e367bd5f781e86bfd2a2e729eec258b7]{
 -- 影の男を思い返していた。^自分を狙って来たのだろうか？^だが、それも終わった事だ。
-I think back on the shadowy^figure. Was he targeting me?^Well, it's not like I'll be^seeing it again.
+// @flowchart_descr
+I think back on that shadowy figure.
+ Was he targeting me?
+ Well, it's not like I'll be seeing him again.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply system_strings/sysmes_text.en:5002]: Went back to doublecheck 08_00_ARC08_8.txt, and it looks like he does say 
> 俺を襲ってきた人影……**あれは男か**……は、[...]

and then refers to them as あの男 a couple times, so I think this is actually fine.

File: system_strings/sysmes_text.en (hunk 6 above)
Comment (rschlaikjer) [reply system_strings/sysmes_text.en:5211]: Hmm, I feel like it does need that 'start out' lead into establish the existence of something to then tip.

File: system_strings/sysmes_text.en (hunk 4 above)
Comment (rschlaikjer) [reply system_strings/sysmes_text.en:5199]: Don't love duplicating worse (hence the expression), also 'gotten' feels like the wrong tense here somehow?

File: system_strings/sysmes_text.en (hunk 8 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply system_strings/sysmes_text.en:2983]: sure

File: system_strings/sysmes_text.en (hunk 9 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply system_strings/sysmes_text.en:3013]: The more careful voice in me says just avoid making references when not needed (and when it stands out too much).

File: system_strings/sysmes_text.en (hunk 10 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply system_strings/sysmes_text.en:5657]: well, I guess it's fine then

File: system_strings/sysmes_text.en (hunk 4 above)
Comment (Petrikow) [reply system_strings/sysmes_text.en:5860]: I remember thinking this one just wasn't very impactful, but it's true that the tense as I've written is a bit weird.
I'm not too torn up about it.

File: system_strings/sysmes_text.en (hunk 6 above)
Comment (Petrikow) [reply system_strings/sysmes_text.en:5872]: I don't really see the need

**Pull request merged**